presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -427,6 +427,7 @@ protected Type toPrestoType(int jdbcType)
             case Types.NVARCHAR:
             case Types.LONGVARCHAR:
             case Types.LONGNVARCHAR:
+            case Types.OTHER:
```

Comment:
If we can (optimistically) map `Types.OTHER` to `varchar`, then maybe we can map _others_ (`default`) to `varchar`? (I'm not suggesting, I'm just thinking aloud and asking for your thoughts)

Either way, what do we do if this turns out to be bad guess, and the `ResultSet.getString` doesn't work for one of the now-seemingly-supported data types?